cpp/include/cudf/strings/split/split.hpp

```diff
@@ -43,18 +43,20 @@ namespace strings {
  *
  * Any null string entries return corresponding null output columns.
  *
- * @param strings_column Strings instance for this operation.
- * @param delimiter UTF-8 encoded string indicating the split points in each string.
+ * @param strings_column Strings instance for this operation
+ * @param delimiter UTF-8 encoded string indicating the split points in each string;
```

Comment:
I'm not a huge fan of the semicolon changes, but don't want to spend too much effort on that discussion.